src/utils/ApiClient.js

```diff
@@ -125,6 +125,12 @@ export default class ApiClient {
      */
     wrapCallback(callback = () => null) {
         return (err, res) => {
+            if (res !== undefined && res.type !== "application/json" && res.type !== "application/jose+json") {
```

Comment:
will the code easier to ready if we do the ff:
`if (res !== undefined && !(res.type === "application/json" || res.type === "application/jose+json")) {`

Reply:
Done